gui/web/src/components/atoms/Button/Button.js

```diff
@@ -34,8 +45,34 @@ class Button extends Component {
     variant: PropTypes.string,
     onClick: PropTypes.func,
     loading: PropTypes.bool,
-    disabled: PropTypes.bool
+    disabled: PropTypes.bool,
+    trackEvent: PropTypes.func,
```

Comment:
what is `trackEvent` -- doesn't seem like it's used anywhere?

Reply:
Honestly, since it's been a few weeks, I just don't remember. I think this would be the function that wraps the event in some format, but I just can't remember. I'll delete this for now. If it matters, we'll find out, and no downside since this won't break anything. How does that sound?

Reply:
perfect.